apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -264,7 +264,21 @@ fields("node") ->
        sc(string(),
           #{ required => true,
              mapping => "emqx.data_dir",
-             desc => "Path to the persistent data directory. It must be unique per broker instance."
+             desc =>
+"""
+Path to the persistent data directory.
+Possible auto-created sub-directories are:
+  - `mnesia/\<node_name>`: EMQX's built-in database directory.
```

Comment:
Just curiosity: why does `<` need to be escaped here?

Reply:
otherwise it would be taken as an html tag.